tests/src/Traits/FixtureTrait.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace Drupal\Tests\minisite\Traits;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Trait FixtureTrait.
```

Comment:
this trait help to create fixture archives - we use it to test multiple permutations of the archive. 

This trait has it's own tests.